test/testdata/configs/wsl.yml

```
@@ -0,0 +1,3 @@
+linters-settings:
+  wsl:
+    tests: 1
```

This seems to be an old setting to include tests or not which is now handled by `golangci-lint`. Let's pick something from the [reference YAML file](https://github.com/golangci/golangci-lint/blob/9da04f5070e67741bb9c98d77c2ee95f13e241df/.golangci.reference.yml#L1796-L1848)

```suggestion
linters-settings:
  wsl:
    allow-cuddle-declarations: false
```